subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/JUnitConsoleLoggingIntegrationTest.groovy

```diff
@@ -41,7 +41,7 @@ class JUnitConsoleLoggingIntegrationTest extends JUnitMultiVersionIntegrationSpe
 
         then:
         outputContains("""
-org.gradle.JUnit4Test > badTest FAILED
+JUnit4Test > badTest FAILED
```

Comment:
❓ Why are these no longer present when executing the test with JUnit 4?

Reply:
This is a multi version spec that executes tests for JUnit 4.12 and JUnit Platform. In case JUnit 4.12 test are executed the package prefix is still present. I will change the test to reflect this.

Reply:
Never mind, they are, we just don't test for them anymore.